WeatherApp/Models/WeatherManager.swift

```diff
@@ -0,0 +1,58 @@
+//
+//  WeatherForecastGetter.swift
+//  WeatherApp
+//
+//  Created by Oleksii Kolomiiets on 5/17/18.
+//  Copyright © 2018 Oleksii Kolomiiets. All rights reserved.
+//
+
+import Foundation
+import CoreLocation
+
+class WeatherManager {
+    static let apiKey = "eceac25e196ca1898edfbfeded3dec64"
+    static let basePath = "https://api.darksky.net/forecast/\(apiKey)/"
+    
+    static func forecast (withLocation location:CLLocationCoordinate2D, completion: @escaping ([LongTimeWeather]?, ShortTimeWeather?, [ShortTimeWeather]?) -> ()) {        
+        let url = basePath + "\(location.latitude),\(location.longitude)"
+        let request = URLRequest(url: URL(string: url)!)
+        
+        let task = URLSession.shared.dataTask(with: request) { (data:Data?, response:URLResponse?, error:Error?) in
+            var dailyForecast:[LongTimeWeather] = []
```

Comment:
`dailyForecast` and `hourlyForecast` are never `nil`, so no necessity in optionals for these parameters in completion closure.